docs/src/extend/custom-processors.md

```diff
@@ -124,3 +124,16 @@ module.exports = {
     }
 }
 ```
+
+You can also use the same custom processor with multiple filename extensions. The following example shows using same processor for both `.md` and `.mdx` files:
+
+```js
+module.exports = {
```

Comment:
It would probably be helpful to include something like this at the top of this example:

```js
const myCustomProcessor = { /* processor methods */ };
```

Just so `myCustomProcessor` is defined within the same example.

Reply:
added